src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/convert_color_i420.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <string>
+#include <vector>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+using ConvertColorI420ParamsTuple = std::tuple<
+        std::vector<InputShape>,                       // Input Shape
+        ov::element::Type,                             // Element type
+        bool,                                          // Conversion type
+        bool,                                          // 1 or 3 planes
+        std::string>;                                  // Device name
+
+class ConvertColorI420LayerTest : public testing::WithParamInterface<ConvertColorI420ParamsTuple>,
+                                  virtual public ov::test::SubgraphBaseTest {
+public:
+    static std::string getTestCaseName(const testing::TestParamInfo<ConvertColorI420ParamsTuple> &obj);
+protected:
+    void SetUp() override;
+};
+
+//----------------------------------------
+
+class ConvertColorI420AccuracyTest : public ConvertColorI420LayerTest {
+protected:
+    // Generate predefined image with R/G/B combinations
+    void generate_inputs(const std::vector<ov::Shape>& targetInputStaticShapes) override;
```

Comment:
Please, update the map which are used in basic `generate_inputs`

Reply:
done